environment.yml

```diff
@@ -1,7 +1,7 @@
 name: weldx
 channels:
-  - defaults
   - conda-forge
+  - defaults
```

Comment:
guess that was the reason I always ended up with the wrong black version every time I recreated my environment

Reply:
that was the reason I changed this 😉